forage-android/src/main/java/com/joinforage/forage/android/pos/ForageTerminalSDK.kt

```diff
@@ -60,21 +64,26 @@ class ForageTerminalSDK(
     }
 
     /**
-     * Securely tokenize a customer's card information
-     * using a ForagePANEditText via UI-based PAN entry
+     * Tokenizes a card via a [ForagePANEdit
+     * Text][com.joinforage.forage.android.ui.ForagePANEditText] Element.
+     * * On success, the object includes a `ref` token that represents an instance of a Forage
+     * [`PaymentMethod`](https://docs.joinforage.app/reference/payment-methods). You can store
+     * the token for future transactions, like to call [checkBalance] or to
+     * [create a Payment](https://docs.joinforage.app/reference/create-a-payment) in Forage's database.
```

Comment:
Not advocating for this change in particular but the current phrasing of 

> you can store the token for future transactions ... in Forage's database.

had me scratching my head a bit because is the token supposed to be stored in Forage's database or the Merchant's database? Not sure if that resonates with you though. If not you can ignore

```suggestion
     * [`PaymentMethod`](https://docs.joinforage.app/reference/payment-methods). You can store
     * the token in your database and reference it for future transactions like to call [checkBalance] or to
     * [create a Payment](https://docs.joinforage.app/reference/create-a-payment).
```

Reply:
I like this much better and will update, thank you!